clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1263,6 +1327,8 @@ protected String getConfigValue(ConfigKey key, String headerName) {
                 return key.type.toString().toLowerCase(Locale.ROOT);
             case "Default":
                 if (key.hasDefault()) {
+                    if (key.alternativeString != null)
```

Comment:
Can we really add this here? `getConfigValue` might be used somewhere else, and it might be expect to get the actual config value back, but never the html-string.

I think we should ensure that the html-string is only returned on the call path of `toHtml`?

Reply:
@mjsax Thanks a lot for your comment. I've removed the constructor and modified the existing constructor. Also I remove the `key.alternativeString` check in `getConfigValue` and put it into `toHtml` instead. Let me know what you think =)